### PR TITLE
visual-studio-code 0.7.0

### PR DESCRIPTION
**Pull request by markhuber**
- Updated link to new url based on javascript (old link broke)
- Updated folder path inside archive (changed from last release)
- Converted to versioned package with sha check

### CODE REVIEW

File: Casks/visual-studio-code.rb [hunk 1]
```diff
@@ -2,13 +2,13 @@
   version :latest
   sha256 :no_check
 
-  url 'https://go.microsoft.com/fwlink/?LinkID=534106'
+  url 'http://go.microsoft.com/fwlink/?LinkID=62088'
```
Comment (scribblemaniac) [inline Casks/visual-studio-code.rb]: This new link does not lead to a download for me.

Comment (markhuber) [inline Casks/visual-studio-code.rb]: Thanks for the catch @scribblemaniac - can you test again, just a typo.

Comment (scribblemaniac) [inline Casks/visual-studio-code.rb]: Yup it works now, however I see that it redirects to https://az764295.vo.msecnd.net/public/0.7.0/VSCode-darwin.zip. Which means we could make this a versioned url (we prefer versioned urls when possible). You could use the versioned url by replacing lines 2-5 with these lines:

    version '0.7.0'
    sha256 '386b8aa79c05a587f25a367fb753ef9c6082389aa384b4f8571a4a7507bbb4ac'
    
    url "https://az764295.vo.msecnd.net/public/#{version}/VSCode-darwin.zip"

**Comment by Ngrd** [comment]
Thank you. Everything looks good. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by scribblemaniac** [comment]
@Ngrd Unbelievable, how many times am I going to forget that :angry:

**Comment by markhuber** [comment]
Squashed. Thanks @Ngrd & @scribblemaniac , first homebrew-cask contrib.

**Comment by Ngrd** [comment]
Thank you @scribblemaniac for all the explanations for this case. @markhuber, your changes were so quick I even hadn't finished my comment about versioned URL in this cask. You both were always one step ahead of me. Well done!

**Comment by Ngrd** [comment]
Thank you for the contribution. It was merged directly as cb1e17b3848e9259fff2af421f3bde40116dd1d8 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**